integ-test/src/test/java/org/opensearch/sql/calcite/standalone/CalcitePPLDateTimeBuiltinFunctionIT.java

```diff
@@ -0,0 +1,1442 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.calcite.standalone;
+
+import static org.opensearch.sql.expression.datetime.DateTimeFunctions.exprYearweek;
+import static org.opensearch.sql.expression.datetime.DateTimeFunctions.formatNow;
+import static org.opensearch.sql.legacy.TestsConstants.*;
+import static org.opensearch.sql.util.MatcherUtils.*;
+import static org.opensearch.sql.util.MatcherUtils.rows;
+
+import java.io.IOException;
+import java.sql.Date;
+import java.time.DayOfWeek;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.TextStyle;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Locale;
+import org.hamcrest.Matchers;
+import org.json.JSONObject;
+import org.junit.Ignore;
+import org.junit.jupiter.api.Test;
+import org.opensearch.client.Request;
+import org.opensearch.sql.data.model.ExprDateValue;
+import org.opensearch.sql.data.model.ExprIntegerValue;
+import org.opensearch.sql.expression.function.FunctionProperties;
+
+public class CalcitePPLDateTimeBuiltinFunctionIT extends CalcitePPLIntegTestCase {
```

Comment:
add IT `CalcitePPLDateTimeBuiltinFunctionPushdownIT`

Reply:
Done

Reply:
Done.